vite.config.ts

```
@@ -32,37 +36,44 @@ export default defineConfig({
+              name: "mui-vendor",
+              test: /node_modules\/(@mui|@emotion)\//,
+            },
+            {
+              name: "charts-maps",
+              test: /node_modules\/highcharts\/.*highmaps/,
+              includeDependenciesRecursively: false,
+            },
+            {
+              name: "charts-sankey",
+              test:
+                /node_modules\/highcharts\/(highcharts-more|modules\/sankey)/,
```

ESLint

[error] Delete `⏎···············`

(prettier/prettier)